pkg/policy/ebpf.go

```diff
@@ -706,6 +829,7 @@ func (ps *policies) computePoliciesConfig() *PoliciesConfig {
 		}
 
 		cfg.EnabledScopes |= 1 << offset
+		cfg.EnabledDataFilters |= 1 << offset
```

Comment:
Shouldn't this be?

```suggestion
		cfg.EnabledDataFilters |= (ExactlyEnabledDataFilters | PrefixEnabledDataFilters | cfg.PostfixEnabledDataFilters) & (1 << offset)
```

Reply:
`cfg.EnabledDataFilters` should hold the bitmap of the policies used instead of holding the bitmap of the enabled data filters.

Now this is replaced by `cfg.EnabledPolicies`